src/tokens/mainnet.json

```diff
@@ -1074,6 +1088,20 @@
       "type": "vault",
       "underlyingTokens": ["0x1ce0a25d13ce4d52071ae7e02cf1f6606f4c79d3"]
     },
+    {
+      "address": "0xDB78B4166580917c9604f8DdfBea5F49B493845c",
+      "decimals": 18,
+      "image": "oribgt-oribgtot.svg",
+      "mintUrl": "https://app.steer.finance/vault/0xDB78B4166580917c9604f8DdfBea5F49B493845c",
+      "name": "oriBGT-oriBGTOT",
+      "protocol": "goldilocks",
```

Comment:
Seems like the protocol is steer
<img width="755" alt="image" src="https://github.com/user-attachments/assets/1a247174-893a-4ba6-9842-034f7a8d3ec8" />